.github/workflows/bench.yml

```diff
@@ -57,6 +57,7 @@ jobs:
       - uses: actions/checkout@v3.0.2
         with:
           fetch-depth: 0
+      - uses: coursier/cache-action@v6
```

Comment:
You'll probably be better off with https://github.com/actions/setup-java?tab=readme-ov-file#caching-sbt-dependencies

I's literally just
```yaml
- uses: actions/setup-java@v4
  with:
    distribution: 'temurin'
    java-version: '21'
    cache: 'sbt'
```

Instead of needing coursier/cache-action and olafurpg/setup-scala